src/Feature/Editor/BufferLineColorizer.rei

```diff
@@ -7,19 +7,24 @@ open Revery;
 open Oni_Core;
 
 /*
- * Type [tokenColor] is a tuple of [(backgroundColor, foregroundColor)]
+ * Type [tokenTheme] is a record containing theme information
  */
-type tokenColor = (Color.t, Color.t);
+type tokenTheme = {
```

Comment:
`tokenTheme` sounds like a theme to me - maybe just `token` (ie, `BufferLineColorizer.token`) or `themedToken` - a token that is themed?

Reply:
Good point. I changed it to `themedToken` in the latest commit!